docs/ports.md

```diff
@@ -0,0 +1,13 @@
+OpenShift Reserved Ports
+========================
+
+OpenShift reserves all ports on hosts that are reserved by [upstream Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/pkg/master/ports/ports.go) as well as the following host ports:
```

Comment:
Can we pin this to a specific release?  Kubernetes might decide to move that file, or they might decide to add or remove entries.  Our implementation will match the state as it stood the last time we looked at that file.  With a pinned link, it would be up to the reader to decide whether that still matched whatever was in Kubernetes current master or not (which they could probably do with `git diff ...`).

Reply:
Hrm.  I'm wondering if we should have an e2e test that enforces this on the cluster (looks at all host network pods or host ports on the cluster and compares it).  Then we don't need this doc.

Reply:
Yeah, I hacked up a conformance test yesterday (after giving up writing a SOCK_DIAG client and just shelling out to `netstat`, lol). However, it's not clear where it belongs, since it tests manifests that are supplied by descendant operators. 

If it's useful, I can add it to openshift/origin.